www/src/data/sidebars/doc-links.yaml

```
@@ -154,6 +154,9 @@
                 - title: Sourcing from Drupal
                   link: /docs/sourcing-from-drupal/
                   breadcrumbTitle: Drupal
+            - title: Sourcing Content from JSON or YAML
+              link: /docs/sourcing-content-from-json-or-yaml/
+              breadcrumbTitle: Sourcing from YAML or JSON
```

```suggestion
              breadcrumbTitle: Sourcing from JSON or YAML
```
Swapping around for consistency.